tools/multicluster/verify.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Verify the signature of a binary with the operator's public key
+# goreleaser takes care of calling this script as a hook.
+
+ARTIFACT=$1
+SIGNATURE="${ARTIFACT}.sig"
+
+HOSTED_SIGN_PUBKEY="https://cosign.mongodb.com/mongodb-enterprise-kubernetes-operator.pem" # to complete
+TMPDIR=${TMPDIR:-/tmp}
+KEY_FILE="${TMPDIR}/host-public.key"
+
+curl -o ${KEY_FILE} "${HOSTED_SIGN_PUBKEY}"
+echo "Verifying signature ${SIGNATURE} of artifact ${ARTIFACT}"
+echo "Keyfile is ${KEY_FILE}"
+
+# When working locally, the following command can be used instead of Docker
+# cosign verify-blob --key ${KEY_FILE} --signature ${SIGNATURE} ${ARTIFACT}
+
+docker run \
+  --rm \
+  -v $(pwd):$(pwd) \
+  -v ${KEY_FILE}:${KEY_FILE} \
+  -w $(pwd) \
+  artifactory.corp.mongodb.com/release-tools-container-registry-local/garasign-cosign \
```

Comment:
same question here

Reply:
This is not a secret per se, although it's true that putting it into a variable might be "cleaner" as this URL is only for internal use

Reply:
I meant for the following reasons:
- secrecy 
- configurability, is there a chance that this might change? If yes its easier to put it in evg

If you don't think thats realistic we can omit this comment

Reply:
I replaced with variables